### DISCUSSION TITLE
$VARIABLES in Snippets, how are they supposed to wok?

### DISCUSSION THREAD

**Discussion by kenNash** [johnlindquist/kit#1508] [Q&A]
Hi Kiters.

This time I'm struggling with the variables ($SELECTION, $CLIPBOARD, etc.) mentioned in the "New Snippet" dialog.

When I trigger this snippet with `,,`,

```
// Name: Wrap
// Snippet: ,,

($SELECTION)$0
```

the current selection is replaced by `($SELECTION)$0`, not by the selection wrapped in brackets.

Am I doing something wrong?

--
Script Kit v2.3.0 on macOS 14.6.1.

**Comment by johnlindquist** [comment]
@kenNash Any chance you can hop on our discord server? I have some later builds ready which may address these issues.

https://discord.gg/DjsGweNA

**Comment by kenNash** [comment]
Oh thank you, now I have v2.23.16 installed.

Now when I type the `,,` trigger, no matter the text selected, a dialog appears that contains "()", with the insertion point at the very end.

1. Why the selection is not inserted between the brackets?
2. Is there a way to skip the dialog, and directly insert the expected result?
3. Thank you!
